modules/geolocation/geolocation.py

```diff
@@ -45,6 +57,10 @@ def create(
             # Image space to camera space
             result, value = camera_intrinsics.camera_space_from_image_space(source[0], source[1])
             if not result:
+                frame = inspect.currentframe()
+                geolocation_logger.error(
+                    f"rotated source vector could not be created for source: {source}", frame
```

Comment:
Capitalize. Same for logging statements below.

Reply:
Sounds good. I noticed in main the logger outpus `main logger initialized` which is why I did all lowercase. Should those be changed as well?